thirdparty/djvulibre/djvulibre-no-locale-mangling.patch

```diff
@@ -0,0 +1,62 @@
+diff --git a/libdjvu/GString.cpp b/libdjvu/GString.cpp
+index b17ed2a..5f844bd 100644
+--- a/libdjvu/GString.cpp
++++ b/libdjvu/GString.cpp
+@@ -88,14 +88,7 @@
+ #endif
+ #include <ctype.h>
+ 
+-#ifndef LC_NUMERIC          //MingW
+-# undef DO_CHANGELOCALE
+-# define LC_NUMERIC 0
+-#endif
+-#ifndef DO_CHANGELOCALE
+-# define DO_CHANGELOCALE 0
```

Comment:
Might be better to just `-DDO_CHANGELOCALE=0` or whatever instead of doing a patch that needs to be maintained? (Although this particular dependency is fairly static.)

Reply:
Yeah, I went with the nuclear option just to be safe (and because I already had a patch to generate anyway).

Reply:
Alright.